### PR TITLE
catalog builder pkg usage updates

### PR DESCRIPTION
**Pull request by aradhakrishnanGFDL**
https://github.com/NOAA-GFDL/analysis-scripts/issues/6

### CODE REVIEW

File: tests/test_freanalysis_clouds.py [hunk 1]
```diff
@@ -47,7 +47,8 @@ def create_data_catalog(path, output="data-catalog"):
     yaml_path = Path(__file__).resolve().parent / "mdtf_timeslice_catalog.yaml"
 
     # Hack to stop click from exiting.
-    command = ["python3", "-m", "scripts.gen_intake_gfdl", str(path), output,
+    # TODO we have to move create catalog as its own pytest in the conda environment perhaps, and avoid this hardcoding
+    command = ["/usr/share/miniconda/envs/analysis-script-testing/bin/python", "-m", "catalogbuilder.scripts.gen_intake_gfdl", str(path), output,
```
Comment (ceblanton) [inline tests/test_freanalysis_clouds.py:51]: When @ilaflott was digging into github CI pipelines last week, he found that one cannot run "conda activate" in the pipeline steps. I didn't understand why.. but you're clearly seeing the same rhing, as you have the `conda activate` commented out, and then use that python's path later.

**Review by ceblanton — CHANGES_REQUESTED** [review]
this is still in draft, but looks good so far!

Installing into conda environments through the github CI pipeline is not as easy as typing the commands into the shell interactively, unfortunately, we're learning.

File: tests/test_freanalysis_clouds.py (hunk 1 above)
Comment (aradhakrishnanGFDL) [reply tests/test_freanalysis_clouds.py:51]: Yeah, it's kinda murky with the init file set up perhaps. We can however activate it, but have to use something like https://github.com/marketplace/actions/setup-miniconda. However, the new pipeline (see new PR) will use the new changes in the catalogbuilder and we will no longer need this particular line.

Comment (ceblanton) [reply tests/test_freanalysis_clouds.py:51]: Do we still want this PR? Sorry for leaving this hanging here. The hard-coded path to the python here would only be used in the CI environments in practice so we could live with it.

Comment (aradhakrishnanGFDL) [reply tests/test_freanalysis_clouds.py:51]: closing this PR as I forgot about this, and there is a new PR now that I got a chance to update this https://github.com/NOAA-GFDL/analysis-scripts/pull/12.

**Pull request closed without merging**